### ISSUE TITLE
Backup important server files

### ISSUE THREAD

**Issue by jmrtns** [cloyne/network#107]
There are currently some files that are not backed up and are also pretty important. These are mostly config files but config files should never be devalued =)

files such as DNS config, OwnCloud config, Sympa config, Router and AP configs, etc. 

Not sure how to best go about this, maybe we can use one of the atom servers as a backup file server, 1TB would probably be much more than enough.

**Comment by mitar** [comment]
server2 has a daily backup of all important files into `/srv/mnt/backup` (a different hard drive set).

Router and AP configs will be stored in nodewatcher.

Only OwnCloud does not have such backup, but you could create one as well.

In general, all configuration should be in Salt anyway. Encrypted with PGP keys, and then stored in GitHub.

**Comment by mitar** [comment]
Or, let me correct this. It had a backup to `/srv/mnt/backup`. Which does not work since `2017-01-28`, when we moved server2 to Salt. I have to make a Docker image for this daily backups.

**Comment by jmrtns** [comment]
Thank you for everything you've done and for continuing to help =) I think at this point I mostly need to work on documenting things since my biggest worry now is future nms being able to do the job or things being able to be easily fixed if an incompetent manager comes in and breaks everything. It seems like I have a bunch of questions that you shouldn't have to answer and that future nms are likely to have also.

**Comment by jmrtns** [comment]
GitHub is a great place for documenting these things but I think the nm exit doc should still have problems/ explanations of everything we have with references to the GitHub since it's sometimes hard to dig through GitHub for answers.

**Comment by mitar** [comment]
Yes. :-) This is why I am explaining all this, how it is, so that we can get this documentation and everything else going.

So you could look into how to store configuration into Salt. This is why I asked you to generate PGP key for network manager. Then we can store that key into Google Drive and have everything bootstrapped from there.

**Comment by jmrtns** [comment]
I foresee 4 types of future nms
1.) doesn't understand servers/networks much, just needs to put things back to working state, won't improve anything, needs everything in baby steps.
2.) over confident, thinks everyone else is an idot and that their implementations are much better, breaks everything and accidentally burns the digital house down when they leave.
3.) well versed in servers/networks, just needs an implementation specific guiding hand to improve the network/servers and will know how to document well
4.) same as 3 but doesn't document well

**Comment by mitar** [comment]
BTW, local backup for server2 is done daily to `/srv/mnt/backup`.
